Assets/Scripts/Game/PlayerMotor.cs

```diff
@@ -491,7 +491,7 @@ public bool GetOnExteriorGroundMethod()
 
             // Must be outside and actually be standing on a terrain object not some other object (e.g. player ship)
             RaycastHit hit;
-            if (GameManager.Instance.PlayerEnterExit.IsPlayerInside || !Physics.Raycast(transform.position, Vector3.down, out hit, rayDistance))
+            if (GameManager.Instance.PlayerEnterExit.IsPlayerInside || !Physics.Raycast(transform.position, Vector3.down, out hit, rayDistance * 2))
```

Comment:
That * 2 triggers splash sounds when horsejumping over water tiles

Reply:
Thanks, I'll get that fixed.

Reply:
I added a fix #1738. I kept the multiplication in place because of a problem when walking on static slopes such as the stairs of Sentinel Castle, which resorted to a grass noise because the raycast didn't hit far enough. I instead made a change to the block that checks groundedness, where instead of checking if the player is currently not using the submerged noise, it will instead check if the player is swimming.